Inventory aggregate filtering backend

ScarpMetal opened the pull request:
These changes allow for aggregating across multiple product types on the Inventory screen.

Frontend: https://github.com/polymerteam/snickerdoodle/pull/286

ScarpMetal committed:
Backend for aggregate inventory filtering

Most notable change is that the serializer now returns process_type and product_types which are a dictionary and an array respectively. This further reflects how ActivityList is structured and will make implementing new features in each dashboard easier in the future.

davidnmora commented on ics/v11/serializers.py:
```diff
@@ -913,59 +913,50 @@ class Meta:
+			'code': item['creating_task__process_type__code'],
+			'unit': item['creating_task__process_type__unit'],
+			'icon': item['creating_task__process_type__icon'],
+			'category': item['creating_task__process_type__category'],
```
I'm curious if there's an existing serializer (or you could write one) which you could pass `item['creating_task__process_type']` into to get just these fields...

davidnmora committed:
Index rankings at :zero:

davidnmora committed:
Write a command to reset all attribute rankings ignoring trashed attrs

davidnmora committed:
Make /move/  new_rank = -1 requests trash the item and remove it from rankings

Giving it a the max possible interger value means you it won't affect attributes if there are fewer than 36,000 Attributes on a single process.

davidnmora committed:
Revert "Backend for aggregate inventory filtering"

This reverts commit e07bac6391fe4b80f711f5aa65e7671b61723c31.

ScarpMetal committed:
Revert "Revert "Backend for aggregate inventory filtering""

This reverts commit 2d446b1a4c21104fc1b893eb2542dd8d020b2ffa.

ScarpMetal committed:
Revert "Backend for aggregate inventory filtering"

This reverts commit e07bac6391fe4b80f711f5aa65e7671b61723c31.

ScarpMetal committed:
Revert "Revert "Backend for aggregate inventory filtering""

This reverts commit 23aa30a86c357eae09d1e77c797e5783d428eeaf.
